src/store/initial-state.js

```diff
@@ -1,19 +1,12 @@
-import sourceCode from "./source-code";
+// import sourceCode from "./source-code";
 
 export default {
   components: {
-    matrix: {
-      data: [{
-        id: 1,
-        project: "linux",
-        githubUser: "torvalds",
-        fileName: "bug.h",
-        contributors: ["rjarzmik", "bjdooks-ct", "arndb"],
-        sourceCode
-      }]
-    },
-    commits: null,
-    positions: null
+    commits: [],
+    positions: [],
+    codeCrawl: {
```

Comment:
Both earth and crawl components rely on initial state being set to null, so please change this. And the codeCrawl property on initialState is no longer needed and can be deleted. 

Otherwise awesome work! :+1: